content/influxdb/clustered/install/deploy.md

```diff
@@ -40,6 +41,9 @@ on your local machine. This allows you to act as the operator would and deploy y
 - You do not want to run the operator in your Kubernetes cluster.
 {{% /note %}}
 
+<!-- Hidden anchor for links to the kubctl/kubit tabs -->
```

Comment:
```suggestion
<!-- Hidden anchor for links to the kubectl/kubit tabs -->
```

Minor typo here/where it's used, but otherwise this LGTM :rocket: Great stuff!